mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule Explorer.MixProject do
   defp deps do
     [
       {:ex_doc, "~> 0.24", only: :docs, runtime: false},
-      {:nx, "~> 0.1.0"},
+      {:nx, "~> 0.1.0", only: :test},
```

Comment:
```suggestion
      {:nx, "~> 0.1.0", optional: true},
```

Reply:
Should be 0.2.0 actually, right?

Reply:
TIL!

Reply:
```suggestion
      {:nx, "~> 0.1", optional: true},
```

It is unlikely the API used by Explorer will ever actually change/break.